Formula/spdlog.rb

```diff
@@ -22,7 +22,10 @@ class Spdlog < Formula
   def install
     ENV.cxx11
 
-    inreplace "include/spdlog/tweakme.h", "// #define SPDLOG_FMT_EXTERNAL", "#define SPDLOG_FMT_EXTERNAL"
+    inreplace "include/spdlog/tweakme.h", "// #define SPDLOG_FMT_EXTERNAL",
+      "#ifndef SPDLOG_FMT_EXTERNAL\n" \
+      "#define SPDLOG_FMT_EXTERNAL\n" \
+      "#endif"
```

Comment:
```suggestion
    inreplace "include/spdlog/tweakme.h", "// #define SPDLOG_FMT_EXTERNAL", <<~EOS
      #ifndef SPDLOG_FMT_EXTERNAL
      #define SPDLOG_FMT_EXTERNAL
      #endif
    EOS
```
This is probably a bit cleaner.

Reply:
Fixed! https://github.com/Homebrew/homebrew-core/commit/265862c51eb80e277cd5ac529c2ad2c1958eb199